bin/add-term

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
```

Comment:
running `add-term` without giving it a term argument should result in a nice error message and the process exiting with exist status 1.